changelog.d/1620.bugfix

```diff
@@ -0,0 +1 @@
+RoomAccessSyncer: Await processing of cached channel modes that changed.
```

Comment:
Suggest making this a misc, unless we can think of a user-facing name of a bug we've fixed?